infra/modules/springapps/springapps.bicep

```diff
@@ -0,0 +1,55 @@
+param asaInstanceName string
+param appName string
+param location string = resourceGroup().location
+param tags object = {}
+param relativePath string
+param databaseUsername string
+@secure()
+param databasePassword string
+param datasourceUrl string
+
+resource asaInstance 'Microsoft.AppPlatform/Spring@2022-12-01' = {
+  name: asaInstanceName
+  location: location
+  tags: tags
+  sku: {
+      name: 'B0'
+      tier: 'Basic'
+    }
+}
+
+resource asaApp 'Microsoft.AppPlatform/Spring/apps@2022-12-01' = {
+  name: appName
+  location: location
+  parent: asaInstance
+  properties: {
+    public: true
+    activeDeploymentName: 'default'
```

Comment:
Seems like `activeDeploymentName` isn't a valid properties in version https://learn.microsoft.com/en-us/azure/templates/microsoft.appplatform/2022-12-01/spring/apps?pivots=deployment-language-bicep.